app/Filament/Resources/HubResource.php

```diff
@@ -57,6 +58,12 @@ public static function infolist(Infolist $infolist): Infolist
                         Infolists\Components\TextEntry::make('title')
                             ->label('Title'),
```

Comment:
??
![image](https://github.com/user-attachments/assets/8ceef23f-b38d-435e-a743-9307b6156af2)

It doesn't appear to be the pipe that makes it angry

Just trying to save the record with a forum_topic_id.

Reply:
In latest, I've lifted the pattern from _NewsCard.tsx_ into the `NoEmoji()` rule and added unit tests for the rule.